File: CabsBooking.Web/ngCabsBooking/KaiHungHuang.CabsBookingSPA/src/app/core/shared/models/booking.ts
import { CabType } from "./cabtype";
import { Place } from "./place";

export interface Booking {
    id?: number;
    email: string;
    bookingDate?: Date;
    bookingTime: string;
    fromPlaceName: string;
    toPlaceName: string;
    pickupAddress: string;
    landmark: string;
    pickupDate?: Date;
    pickupTime: string;
    cabTypeName: string;
    contactNo: string;
    status: string;
    fromPlace?: Place;
    toPlace?: Place;
    cabTypeId?: number;
    cabType?: CabType;
}

File: CabsBooking.Web/KaiHungHuang.CabsBooking.Core/Entity/Booking.cs
using System;
using System.Data.Common;

namespace KaiHungHuang.CabsBooking.Core.Entity
{
    public class Booking
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public DateTime? BookingDate { get; set; }
        public string BookingTime { get; set; }
        public int? FromPlace { get; set; }
        public int? ToPlace { get; set; }
        public string PickupAddress { get; set; }
        public string Landmark { get; set; }
        public DateTime? PickupDate { get; set; }
        public string PickupTime { get; set; }
        public int? CabTypeId { get; set; }
        public string ContactNo { get; set; }
        public string Status { get; set; }
        public Place ToPlaceName { get; set; }
        public Place FromPlaceName { get; set; }
        public CabType CabType { get; set; }
    }
}